tests/foreman/ui/test_syncplan.py

```diff
@@ -124,7 +124,7 @@ def test_positive_end_to_end(session):
         assert syncplan_values['details']['description'] == new_description
         # Delete sync plan
         session.syncplan.delete(plan_name)
-        assert not session.syncplan.search(plan_name)
+        assert 'Your search returned zero Sync Plans.' in str(session.syncplan.search(plan_name))
```

Comment:
`empty_search message = 'Your search returned zero Sync Plans.'` at module level and use across other tests if it is common. Just suggestion.

Reply:
Do you mean something like:

    @fixture(scope='module')
    def module_org():
    global empty_search_message
    empty_search_message = 'Your search returned zero Sync Plans.'
    return entities.Organization().create()